src/core_read.cpp

```diff
@@ -40,8 +39,9 @@ CScript ParseScript(const std::string& s)
                 continue;
             mapOpNames[strName] = static_cast<opcodetype>(op);
             // Convenience: OP_ADD and just ADD are both recognized:
-            boost::algorithm::replace_first(strName, "OP_", "");
```

Comment:
Wouldn't `find` instead of `rfind` make the code match the previous `replace_first` version 100%?

Reply:
Indeed, to match the previous version 100% the code would look something like this:
```C++
    ...
    // Convenience: OP_ADD and just ADD are both recognized:
    auto pos = strName.find("OP_");
    if (pos != std::string::npos) {
        strName = strName.substr(0, pos) + strName.substr(pos+3);
        mapOpNames[strName] = static_cast<opcodetype>(op);
    }
    ...
```
That would also eliminate an "OP_" substring that is _not_ at the start of `strName`. Don't know though if that really makes sense to match the previous version 100% (which did more than needed)? I guess it's a matter of taste and how strict a refactoring is defined.